admin/starter/.github/workflows/close-pull-request.yml

```diff
@@ -2,17 +2,24 @@ name: Close Pull Request
 
 on:
   pull_request_target:
-    types: [opened]
+    types: [opened, reopened]
+
+permissions:
+  pull-requests: write
+
+env:
```

Comment:
No. This is not what I meant. You can still retain this as step env below. No need to make them as global env.